api/src/test/java/org/apache/iceberg/expressions/TestAggregateBinding.java

```diff
@@ -18,44 +18,28 @@
  */
 package org.apache.iceberg.expressions;
 
-import java.util.Arrays;
 import java.util.List;
 import org.apache.iceberg.exceptions.ValidationException;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableList;
 import org.apache.iceberg.types.Types;
 import org.apache.iceberg.types.Types.StructType;
 import org.assertj.core.api.Assertions;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class TestAggregateBinding {
-  private static final List<Expression.Operation> AGGREGATES =
-      Arrays.asList(Expression.Operation.COUNT, Expression.Operation.MAX, Expression.Operation.MIN);
+  private static final List<UnboundAggregate> list =
```

Comment:
I noticed that in this entire file we're using `UnboundAggregate` without specifying a generic type. I think it would be good to replace all `UnboundAggregate` by `UnboundAggregate<String>`? Same for `BoundAggregate`, which should probably be `BoundAggregate<String, String>`

Reply:
Fixed. Thanks! I accidentally put `String` type in the factory methods. It should be `T`. I fixed that.